src/components/ContentSidebar/ActivityFeed/activity-feed/activityFeedUtils.js

```diff
@@ -45,15 +50,17 @@ export function collapseFeedState(feedState: ?FeedItems): FeedItems {
                 [prevModifiedBy.id]: { ...prevModifiedBy },
             };
 
+            const modifiedBy = tmpCurModifiedBy || PLACEHOLDER_USER;
```

Comment:
This definitely works, but could we potentially use a user with no ID? It's slightly less accurate to use the placeholder user since they get counted as anonymous.

Reply:
If we do stick with the anonymous user, how does that look in the UI when they upload/modify a new version?

Reply:
shows "__ uploaded a version" which is the same as Amsterdam currently so leaving it as is.